src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,18 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final Color[] COLORS = Color.values();
+    private static final int COLORS_AMOUNT = COLORS.length;
```

Comment:
```suggestion
```
not worth it, you can get length from your first constant at any time with no cost